lib/code_corps/model/task.ex

```diff
@@ -58,6 +60,7 @@ defmodule CodeCorps.Task do
     |> changeset(params)
     |> cast(params, [:status])
     |> validate_inclusion(:status, statuses())
+    |> set_closed_at()
```

Comment:
I understand why you placed this in update but I do think since it's possible we'll need to write another issue to address this with GitHub sync. Can you open such an issue? We'd need to handle the closed event on GitHub and propagate that close to here. This could happen on create so I want to be sure we're not using the create changeset here.

Reply:
Sure raised an issue. But I am unaware of GithubSync feel free to edit the issue -https://github.com/code-corps/code-corps-api/issues/992